### PR TITLE
Improve overall accessibility in superdesk

### PR DESCRIPTION
**Pull request by pavlovicnemanja**

### CODE REVIEW

File: scripts/apps/authoring/views/article-edit.html [hunk 1]
```diff
@@ -524,17 +524,18 @@
...
      sd-width="{{editor.sign_off.sdWidth|| 'full'}}">
-    <label class="field__label">{{label('sign_off')}}</label>
+    <label for="sign_off" class="field__label">{{label('sign_off')}}</label>
     <button
         id="sign-off-unlock"
         class="icn-btn btn-signOff"
         ng-click="changeSignOffEdit()"
         tabindex="{{editor.sign_off.order}}"
-        title="Unlock edit Sign-Off"
+        title="Toggle Sign-Off lock"
```
Comment (tomaskikutis) [inline scripts/apps/authoring/views/article-edit.html:533]: Title should be translated too.

File: scripts/apps/monitoring/views/monitoring-view.html [hunk 2]
```diff
@@ -182,7 +184,11 @@ <h3 class="subnav__page-title sd-flex-no-grow sd-empty">
                         <div sd-item-sortbar class="sortbar-container"></div>
 
                         <div class="dropdown dropdown--align-right" dropdown ng-if="type === 'monitoring'">
-                            <button class="navbtn dropdown__toggle" dropdown__toggle><i ng-class="{'icon-list-view': !monitoring.viewColumn && (!view || view === 'compact'), 'icon-grid-view': view === 'photogrid', 'icon-kanban-view': monitoring.viewColumn}"></i></button>
+                            <button area-label="{{!monitoring.viewColumn && (!view || view === 'compact') ? 'List View' : view === 'photogrid' ? 'Photo Grid' : monitoring.viewColumn ? 'Kanban View : ''}}"
```
Comment (tomaskikutis) [inline scripts/apps/monitoring/views/monitoring-view.html:187]: translation needed

File: scripts/apps/search/views/item-sortbar.html [hunk 3]
```diff
@@ -1,19 +1,22 @@
 <div class="sortbar" ng-show="canSort()">
     <span ng-show="total" translate>Total:</span>
     <span ng-show="total" class="label-total">{{total}}</span>    
-    <div class="dropdown dropdown--align-right dropdown--hover" dropdown>
-        <button id="order_button" class="dropdown__toggle" dropdown__toggle>
+    <div class="dropdown dropdown--align-right" dropdown>
+        <button id="order_button" class="dropdown__toggle" dropdown__toggle aria-label="{{ 'Sort items' | translate }}">
```
Comment (tomaskikutis) [inline scripts/apps/search/views/item-sortbar.html:5]: Shouldn't aria label match with text content inside the button?

File: scripts/apps/workspace/views/workspace-dropdown.html [hunk 4]
```diff
@@ -1,3 +1,4 @@
+<h1 id="workspace-heading" class="a11y-only"><span translate>Workspace:</span> {{ selected.name | translate}}</h1>
```
Comment (tomaskikutis) [inline scripts/apps/workspace/views/workspace-dropdown.html:1]: > `{{ selected.name | translate}}`

Workspace names are coming from user input so there's no use translating it using strings extracted from the code

File: scripts/core/menu/styles/menu.scss [hunk 5]
```diff
@@ -499,150 +505,125 @@
     top: 0;
 }
 
-#main-menu {
+.main-menu
```
Comment (tomaskikutis) [inline scripts/core/menu/styles/menu.scss:508]: you forgot to remove this

File: scripts/apps/authoring/views/authoring-header.html [hunk 6]
```diff
@@ -181,10 +183,11 @@
                  sd-width="{{editor.place.sdWidth}}"
                  ng-attr-title="{{schema.place.readonly === true ? readOnlyLabel : ''}}"
                  order="{{editor.place.order}}">
-                <label class="authoring-header__item-label">{{getLabelForFieldId('place')}}</label>
+                <label class="authoring-header__item-label" for="place">{{getLabelForFieldId('place')}}</label>
                 <div class="authoring-header__input-holder">
                     <div sd-meta-terms ng-if="!features.places_autocomplete && !single_value.locators"
                          class="data"
+                         id="place"
```
Comment (tomaskikutis) [inline scripts/apps/authoring/views/authoring-header.html:190]: It'd be good to use unique strings for identifiers in order to be able to find usages easily. If you grep for `place` or `title` there are too many matches and it's hard to see usages. We could add `a11y-` to IDs. Like `id="a11y-place"`.

File: scripts/apps/search/views/item-sortbar.html (hunk 3 above)
Comment (fritzSF) [reply scripts/apps/search/views/item-sortbar.html:5]: In this case, it's there to give more context to the action. The text in the button is accessible anyway, the aria-label is there as a description. I'll probably change the logic though and make the context visible to all users.

File: scripts/apps/monitoring/views/monitoring-view.html [hunk 7]
```diff
@@ -184,7 +184,7 @@ <h2 class="subnav__page-title sd-flex-no-grow" ng-hide="type === 'monitoring'">
                         <div sd-item-sortbar class="sortbar-container"></div>
 
                         <div class="dropdown dropdown--align-right" dropdown ng-if="type === 'monitoring'">
-                            <button area-label="{{!monitoring.viewColumn && (!view || view === 'compact') ? 'List View' : view === 'photogrid' ? 'Photo Grid' : monitoring.viewColumn ? 'Kanban View : ''}}" 
+                            <button area-label="{{!monitoring.viewColumn && (!view || view === 'compact') ? gettext('List View') : view === 'photogrid' ? gettext('Photo Grid') : monitoring.viewColumn ? gettext('Kanban View') : ''}}"
```
Comment (tomaskikutis) [inline scripts/apps/monitoring/views/monitoring-view.html:187]: Translation string extraction using `gettext` doesn't work from `.html` files - either translate attribute or filter should be used.

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**